### ISSUE TITLE
 Connecting external users via their proxy

### ISSUE THREAD

**Issue by Tam-Ov** [jitsi/docker-jitsi-meet#1869]
Hi.
We have server behind a NAT.
Added lines 
org.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=192.....
org.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS=18.....

External users are connecting normally. But there are problems for those external users who use a proxy. They see and hear only themselves. 
Wireshark shows that traffic on port 10000 does not go through a proxy, there is an attempt to connect to external ip address.

Internet Protocol Version 4, Src: 192.168.1.26, Dst: 18X.....202
User Datagram Protocol, Src Port: 54052, Dst Port: 10000

Help me understand why data on port 443 goes through a proxy, but data on port 10000 does not

**Comment by saghul** [comment]
What type of proxy are they using? Any chance they are blocking UDP traffic?

**Comment by Tam-Ov** [comment]
They using squid. 
In squid allowed all traffic to udp 10000. 443 goes through squid. 10000 port goes through the computer’s gateway, not through squid. Need to additionally open 10000 port in firewall of a third-party organization

**Comment by saghul** [comment]
Yes squid is an HTTP proxy, so they need to allow traffic to UDP port 10000 on the outbound.
